lbz/dev/server.py

```diff
@@ -55,17 +55,20 @@ def _get_route_params(self, org_path: str) -> Tuple[Union[str, None], Union[dict
                         return org_route, params
         return None, None  # to be raised
 
-    def _send_json(self, code, obj):
+    def _send_json(self, code, obj, headers=None):
         # Make sure only one response is sent
         if self.done:
             return
 
         self.send_response(code, message=None)
-        self.send_header("Content-Type", "application/json;charset=UTF-8")
+
+        if not headers:
+            headers = {"Content-Type": "application/json;charset=UTF-8"}
```

Comment:
I am not sure if this application/json should be implicitly sent in this case, but I leave decision for you.